resources/js/components/UserEditForm.vue

```diff
@@ -267,12 +297,21 @@ export default {
         .put(this.dataUrl, this.user)
         .then(response => {
           this.hasError = false;
-          if (this.user.clickup_email !== this.clickUpEmailInDatabase && this.user.clickup_email && this.user.clickup_email.length > 0) {
+          if ((this.user.clickup_email !== this.clickUpEmailInDatabase && this.user.clickup_email && this.user.clickup_email.length > 0) 
+          && (this.user.autodesk_email !== this.autodeskEmailInDatabase && this.user.autodesk_email && this.user.autodesk_email.length > 0)) {
+            this.clickUpEmailInDatabase = this.user.clickup_email;
+            this.autodeskEmailInDatabase = this.user.autodesk_email;
+            this.feedback = 'Saved! Look out for an email from ClickUp and Autodesk in the next few minutes.'
```

Comment:
```suggestion
            this.feedback = 'Saved! Look out for emails from ClickUp and Autodesk in the next few minutes.'
```

Reply:
Gosh dangit brain. Thank you for catching this